app/js/disease-associations-table-directive.js

```diff
@@ -339,75 +359,91 @@ angular.module('cttvDirectives')
                 // First make a call to know how many rows there are:
                 var optsPreFlight = {
                     disease: scope.disease,
+                    target: scope.target,
```

Comment:
I forgot to take this one out. Could you please change it to?
if (scope.target) {
   optsPreFlight.target = scope.target
}